### PR TITLE
Initializing project with generated scaffolding.

### PR DESCRIPTION
**Pull request by tbiethman**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Cerner Corporation
+
+- {First Name} {Last Name} [@{github username}]
```
Comment (mjhenkes) [inline CONTRIBUTORS.md]: add your name :)

Comment (tbiethman) [inline CONTRIBUTORS.md]: Argh

**Comment by ryanthemanuel** [comment]
👍

**Comment by bjankord** [comment]
+1

**Pull request merged**